### PR TITLE
NR-112249 - updating MExN dashboard and alerts

### PR DESCRIPTION
**Pull request by pkosanam**
Ticket: https://issues.newrelic.com/browse/NR-112249

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/mexn/MongoDBUp.yml [hunk 1]
```diff
@@ -19,20 +19,11 @@ valueFunction: SINGLE_VALUE
...
     # Operator used to compare against the threshold.
-    operator: ABOVE
+    operator: NOT EQUAL TO
     # Value that triggers a violation
-    threshold: 2000
+    threshold: 1
     # Time in seconds; 120 - 3600
-    thresholdDuration: 300
-    # How many data points must be in violation for the duration
-    thresholdOccurrences: ALL
```
[the remark this answers is not in the document]
Comment (pkosanam) [reply alert-policies/mexn/MongoDBUp.yml:28]: Hi @brammerl ,
I have now added the warning condition too. But I still see that the "Validation / Validate Quickstart Schema" check did not pass yet.
Thank you 😃

File: alert-policies/mexn/MongoDBUp.yml [hunk 2]
```diff
@@ -19,20 +19,20 @@ valueFunction: SINGLE_VALUE
 terms:
   - priority: CRITICAL
     # Operator used to compare against the threshold.
-    operator: ABOVE
+    operator: NOT EQUAL TO
```
Comment (brammerl) [inline alert-policies/mexn/MongoDBUp.yml:22]: This must be either ABOVE, BELOW or EQUALS

File: alert-policies/mexn/MongoDBUp.yml [hunk 3]
```diff
@@ -19,20 +19,20 @@ valueFunction: SINGLE_VALUE
...
+    threshold: 1
     # Time in seconds; 120 - 3600
-    thresholdDuration: 300
+    thresholdDuration: 240
     # How many data points must be in violation for the duration
     thresholdOccurrences: ALL
   - priority: WARNING
     # Operator used to compare against the threshold.
-    operator: ABOVE
+    operator: NOT EQUAL TO
```
Comment (brammerl) [inline alert-policies/mexn/MongoDBUp.yml:31]: Must be ABOVE, BELOW or EQUALS

File: alert-policies/mexn/MongoDBUp.yml (hunk 2 above)
Comment (pkosanam) [reply alert-policies/mexn/MongoDBUp.yml]: Thank you @brammerl, It worked.. 👍

File: alert-policies/mexn/MongoDBUp.yml (hunk 3 above)
Comment (pkosanam) [reply alert-policies/mexn/MongoDBUp.yml]: Thank you @brammerl, It worked.. 👍

**Review by josephgregoryii — DISMISSED** [review]

**Comment by josephgregoryii** [comment]
@seemantk - this looks good from our end, wrapping you in for content approval 👋🏻

**Review by mdumpati — APPROVED** [review]
Looks good to me

**Review by seemantk — APPROVED** [review]
LGTM

**Pull request merged**